tests/unit/app/validations/harvest_grow_test.py

```diff
@@ -0,0 +1,12 @@
+from app.validation.harvest_grow import validate_harvest_grow
+from unittest.mock import MagicMock
+import pytest
+
+def test_harvest_grow():
+    AppConfig = MagicMock()
+    assert validate_harvest_grow(AppConfig, {"grow": {"grow_id": 4}}) == (True, "")
+
+def test_harvest_grow_fail():
+    AppConfig = MagicMock()
```

Comment:
can you validate that methods on the database object are called here? For example, some of the db methods that harvest a grow by doing DB writes. Pytest allows you to check what functions were called, how many times, with what arguments etc

Reply:
You can also mock out the return value to make it return different kinds of things, and test certain scenarios.

Reply:
+1 on using mocks